napari/components/layerlist.py

```diff
@@ -62,6 +62,41 @@ class LayerList(SelectableEventedList[Layer]):
     selection.events._current : (value: _T)
         emitted when the current item has changed. (Private event)
 
+    Notes
+    -----
+
+    Note that ``changed`` events are only emitted when the value of an item in
+    the layer list item value is changed, not when the item is inserted or removed.
```

Comment:
```suggestion
    Note that ``changed`` events are only emitted when a change occurs to the value of an item in
    the layer list. A ``changed`` event is not emitted when the item is inserted or removed from the layer list.
```

Reply:
You know, this suggestion made me think: it's not when the value changes, it's when the *object reference* changes. For example, `layerlist[idx].scale = [2, 1, 1]` changes the "value" of the layer at position `idx`, but it does not cause `changed` to be emitted.

So, maybe we can deprecate the changed event (or make it what people expect!), and rename it to `replaced`?

Reply:
Good point @jni. I updated the suggested comment above to use your wording.

Reply:
Added a future placeholder task to revisit during an event system refactor.